packages/angular/src/schematics/init/init.spec.ts

```diff
@@ -1,21 +1,25 @@
-import { Tree } from '@angular-devkit/schematics';
-import { runSchematic, callRule } from '../../utils/testing';
-import { createEmptyWorkspace } from '@nrwl/workspace/testing';
-import { readJsonInTree, updateJsonInTree } from '@nrwl/workspace';
+import { readJson, Tree } from '@nrwl/devkit';
+import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing';
+import { Linter } from '@nrwl/workspace';
+
+import init from './init';
+import { E2eTestRunner, UnitTestRunner } from '../../utils/test-runners';
 
 describe('init', () => {
-  let appTree: Tree;
+  it('should add angular dependencies', async () => {
+    // ARRANGE
+    const host = createTreeWithEmptyWorkspace();
```

Comment:
This could be done in a `beforeEach` to avoid repeating it in every test case.

Reply:
I did have it in a `beforeEach` to start with, but I feel like this might be a little bit more explicit. 

You can go to any test and see the set up easily.

I guess, all the setup is roughly the same though, so it could be just as well to have it in `beforeEach`

Reply:
Yeah, I agree with being explicit on tests. I'm against reusing functions to test stuff.

For this case though, I don't think being explicit gives you much. All generator tests (not just this generator) will always need to create a tree as part of the setup. I feel it's just the additional setup needed by individual tests, what really gives you the sense of what's being set up.